### ISSUE TITLE
WinSetBak.ps1 InvalidOperation: cannot call a method on a null-valued expression.

### ISSUE THREAD

**Issue by frosit** [LesFerch/WinSetView#129]
When running `WinSetBak.ps1` using Powershell version 7.4.5 I get the following error:

```shell
1. Select backup folder (Current: C:\Users\xxxxxxxxx\Applications\WinSetView-main\Tools\WinViewBak)
2. Create a backup of current Explorer folder views
3. Restore Explorer folder views from backup file
4. Clear current Explorer views and return to Windows defaults

Choose option or press Enter to Exit: 2

InvalidOperation: C:\Users\xxxxxxxxx\Applications\WinSetView-main\Tools\WinSetBak.ps1:155
Line |
 155 |        Cmd /c Copy $FileList.Trim('+') $BakFile >$Null 2>$Null
     |        ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
     | You cannot call a method on a null-valued expression.

Backup complete.

Press Enter to continue...:
```

While running the same script using Powershell 5.1.22621.4249 results in a backup file in the WinViewBak directory. My guess is that powershell 7 is not supported here

**Comment by LesFerch** [comment]
That's correct. It was only tested with PowerShell 5.1. When I have a moment, I'll see what's needed to make it work with 7. I can't imagine it's anything too difficult.

**Comment by LesFerch** [comment]
This is more weird than I thought. Reg.exe works perfectly fine from a PS 7.4.5 prompt, but when run from my script, it reports that the registry key/value was not found, like it's running in a sandbox when executed in a script. Pulling my hair out here and ChatGPT is more confused than I am. If you have any insight, I'd sure appreciate some help.

**Comment by LesFerch** [comment]
It turns out to be simple, but subtle. Here's an example:

`reg export '"HKCU\Software\Classes\Local Settings\Software\Microsoft\Windows\Shell\BagMRU"' test.reg /y`

The additional inner `"` characters are unnecessary in both PS 5.1 and PS Core , but I had included them, erroneously assuming they're needed because of the space in the path. However, the code above still works fine in PS 5.1 (maybe it automatically strips the extra quotes) but fails in PS Core.

**Comment by LesFerch** [comment]
Tools\WinSetBak.ps1 now works on PS Core. Distributions files have been updated.
